mocks/service/github/index.js

```diff
@@ -21,7 +21,7 @@ router.post('/login/oauth/access_token', responses.okayAuthenticated);
 
 router.get(
   '/repos/:owner/:name/contents/snapcraft.yaml',
-  responses.okaySnapcraftYamlFound
+  responses.errorSnapcraftYamlNotFound
```

Comment:
We want that to be default mock response for now?
Probably not worth to make it conditional ATM depending on which mocked repo is sent in params...